fix/w-11280374

PabloCamino opened the pull request:
This should prevent us from repeating investigations like w-11275069.
Feel free to edit style ad language as you know how to do ;-)

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

PabloCamino committed:
Added agent version mismatch secrion

This should prevent us from repeating investigations like w-11275069.
Feel free to edit style ad language as you know how to do ;-)

Commit:
edit

IsaacEldridge commented on modules/ROOT/pages/troubleshoot-guide.adoc:
```diff
@@ -384,3 +384,24 @@ kubectl -n monitoring delete pod <influxdb_pod_name>
+
+=== Failed Agent Upgrades
+
+In this situation, an agent upgrade fails, and you roll it back, but the desired (upgraded) version is the version listed in Anypoint Platform. To correct this issue, apply the procedure described in https://help.mulesoft.com/s/article/Degrade-Status-as-Runtime-Fabric-Agent-Version-Mismatch-Detected[Degrade Status as Runtime Fabric Agent Version Mismatch Detected^].
```
@PabloCamino the ^ at the end of the link adds an external link symbol when the doc is published. Just FYI

PabloCamino replied:
Thanks for letting me know! I changed the wording, please have a look at the latest changes. Thank you @IsaacEldridge .

IsaacEldridge reviewed (commented):
@PabloCamino can you please review my edits

Commit:
edit

PabloCamino committed:
Changed to indirect wording

The customer does not roll back the upgrade, it eventually gets rolled back by helm.
I changed the wording for that.

Commit:
Merge branch 'v1.13' into fix/w-11280374

Commit:
edits

JennyHajee commented on modules/ROOT/pages/troubleshoot-guide.adoc:
```diff
@@ -384,3 +384,24 @@ kubectl -n monitoring delete pod <influxdb_pod_name>
+
+In this situation, a Runtime Fabric agent upgrade fails and gets rolled back, but the desired (upgraded) version is listed in Anypoint Platform. To correct this issue, apply the procedure described in https://help.mulesoft.com/s/article/Degrade-Status-as-Runtime-Fabric-Agent-Version-Mismatch-Detected[Degrade Status as Runtime Fabric Agent Version Mismatch Detected^].
+
+=== Resotration of Pre-Upgrade Backups
```
```suggestion
=== Restoration of Pre-Upgrade Backups
```

Commit:
typo

Co-authored-by: Jenny Hajee <94015849+JennyHajee@users.noreply.github.com>

Commit:
typo

Commit:
Merge branch 'v1.13' into fix/w-11280374
